gem/constants.py

```diff
@@ -0,0 +1,3 @@
+MALE = "m"
+FEMALE = "f"
+GENDERS = {(MALE, "male"), (FEMALE, "female")}
```

Comment:
After discussion on Slack I've added Trans as an option and indicated when registering that your gender will only be visible to you.